src/test/resources/application.conf

```diff
@@ -2,4 +2,5 @@ akka {
   loggers = ["akka.testkit.TestEventListener"]
   loglevel = DEBUG
 }
+akka.stream.materializer.max-input-buffer-size = 256
```

Comment:
Is this really needed? Your test reproduced the problem with just 25 messages.

Reply:
yes, this is needed, default buffer size is 16, so i couldn't publish more without consuming them

Reply:
hmm however we are blocking for the size >10 in the test
let me try to remove it